### ISSUE TITLE
Using with periods_per_year = 52

### ISSUE THREAD

**Issue by MarcosBizont** [Lumiwealth/quantstats_lumi#26]
I am wondering if anybody is using this library with periods parameters different than 365. 

I am finding multiple issues (Sortino is all over the place), but before I put together a patch, I want to know if anybody else is experiencing similar problems.

Thanks

**Comment by grzesir** [comment]
What issues are you getting? And why would you change the period from 365?

Robert Grzesik 347-635-3416

**Comment by MarcosBizont** [comment]
I am using a strategy that is rebalanced every week. As a result, my data series is provided by week instead of daily, as stock returns usually are.

Until now, I have detected the CAGR being wrongly calculated if the period is not 365 and I am currently working in the Sortino.

**Comment by grzesir** [comment]
Makes sense. If you send a pull request to fix this issue I can merge it
into the library and release the fix

Robert Grzesik 347-635-3416

**Comment by SamAuclair** [comment]
Hi @MarcosBizont ,
I am using a period of 52 as well and am encountering some problems.

for example, in the cagr() function from the stats.py module, I believe that this line:
years = (returns.index[-1] - returns.index[0]).days / periods

should be:
years = (returns.index[-1] - returns.index[0]).days / 365

**Comment by MarcosBizont** [comment]
Hello!

Thank you for your email. I am currently out of the office and will be
returning on November 25. During this time, I will have limited access to
email. If your matter is urgent, please
***@***.*** ***@***.***> Otherwise, I
will respond to your message as soon as possible upon my return.

Thank you for your understanding.
Best regards
